journal/updated/01-journal.adoc

```diff
@@ -767,251 +767,270 @@ nevertheless, afterward it yields the peaceable fruit of righteousness,
-if you will go with us in our way, and give us bread to eat, and raiment to put on, then,
-O Lord, you shall be our God; and the Lord was with us in all our journey,
-and gave unto us his sweet and comfortable presence.
-
-Soon after we came to Welch-Pool, those professors, who had been and were in great power,
-began to be fainthearted, because of the report of bringing in king Charles the second;
-which in a little time was accomplished, and those that were in great pomp,
+where we believed the Lord desired us to be; and we said, "`O Lord,
+if You will go with us in our way, and give us bread to eat,
+and raiment to put on, then, O Lord, You shall be our God;`"
+and the Lord was with us during all of our journey,
+and gave us His sweet and comfortable presence.
+
+Soon after we came to Welch-Pool,
+the professors who had been in great power
+began to be fainthearted because of the report of bringing in king Charles the second,
+which in a little time was accomplished, and those that were in great pomp
 were brought to prison themselves.
-And I was had before the first justices that were made in
+I was brought before the first justices that were made in
 these parts by the authority of king Charles the second,
 in the year 1660, notwithstanding I was a prisoner to the magistrate of the town.
-When I went up before them, many of the people of the town followed me,
+When I went up before them, many of the people of the town followed me
 to see what would become of me, and to what prison they would send me,
 or what punishment they would inflict upon me.
 But the Lord was with me, and I feared not man, whose breath is in his nostrils,
 but the living God, whom I desired to obey in all things.
 
-When I was come into the room, it being in the night, the high sheriff, colonel Mostyn,
-and the justices stood as people in amaze,
-to see me come with my hat on my head among them,
+When I had come into the room (it being in the night),
+the high sheriff, colonel Mostyn, and the justices stood in amazement
+to see me come in among them with my hat on my head,
 and spoke not one word to me for some time.
-In a little while, I asked them, whether they sent for me there; they said they did.
-
-One of the justices asked me, where I had that new way, and strange religion.
-I answered him,
-It was the good old way that the prophets and servants of God lived and walked in;
-and that way I had found, and desired to walk in it all my days.
-That justice was peevish, and said, I think the man is mad:
-I think we must have him whipped; though I answered them according to scripture,
-yet they were ignorant of it.
-They demanded of me to take the oath of allegiance and supremacy; I told them,
-that my Lord and Master, Christ Jesus, and his apostle James,
+In a little while, I asked them whether they sent for me there; they said they did.
+
+One of the justices asked me where I had acquired that new and strange religion.
```

Comment:
I think you need to keep the word "way" here, where it says "that **new way** and strange religion", because he refers to the word "way" it below. 
And I would maybe reword this a bit, so that it reads: **I told him it was the good old way which I had found, the same that the prophets and servants of God lived and walked in, and I desired to walk in it all my days.**